### PR TITLE
macvim.org

### LINKED ISSUE #746: +macvim (545/548)
Formulae: https://formulae.brew.sh/formula/macvim

### PR DESCRIPTION
**Pull request by and-ri**
closes #746

### CODE REVIEW

File: projects/macvim.org/package.yml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+dependencies:
+  cscope.sourceforge.io: '*'
+  lua.org: '*'
+  python.org: ^3.11
+  ruby-lang.org: '*'
+  invisible-island.net/ncurses: '*'
+runtime:
+  env:
+    PYTHONHOME: "{{deps.python.org.prefix}}"
+    DYLD_LIBRARY_PATH: "{{deps.python.org.prefix}}/lib/libncurses.6.dylib:$DYLD_LIBRARY_PATH"
```
Comment (jhheider) [inline projects/macvim.org/package.yml]: This doesn't look right; wouldn't it be `deps.invisible-island.net/ncurses/prefix`? I also worry about setting `PYTHONHOME`.

Comment (and-ri) [reply projects/macvim.org/package.yml]: oh `DYLD_LIBRARY_PATH` it's my mistake 😄 

without `PYTHONHOME` I have:

```
Could not find platform independent libraries <prefix>thon3'
Could not find platform dependent libraries <exec_prefix>
Python path configuration:
  PYTHONHOME = (not set)
  PYTHONPATH = (not set)
```

**Comment by and-ri** [comment]
I hope #3121 helps.

**Comment by and-ri** [comment]
hmm... strange
Perhaps the changes (#3121) will take effect later?

**Comment by jhheider** [comment]
I don't see a dep on ncurses. Could that be the issue?

**Comment by jhheider** [comment]
You probably need to call fix-machos manually on the app dir so it updates the library paths. See fixup.ts for invocation (or search for fix-machos.rb; I think some other packages might call it manually too).

**Comment by jhheider** [comment]
bam

**Pull request merged**